lib/experiments-page.php

```diff
@@ -3,248 +3,790 @@
+			'title'       => __( 'Data Views: new media modal', 'gutenberg' ),
+			'description' => __( 'Enables a new media modal experience powered by Data Views for improved media library management.', 'gutenberg' ),
+			'category'    => 'data-views',
+		),
+		array(
+			'id'          => 'gutenberg-full-page-client-side-navigation',
+			'title'       => __( 'Interactivity API: Full-page client-side navigation', 'gutenberg' ),
+			'description' => __( 'Enables full-page client-side navigation, powered by the Interactivity API.', 'gutenberg' ),
+			'category'    => 'interactivity',
+		),
+		array(
+			'id'          => 'gutenberg-content-only-pattern-insertion',
+			'title'       => __( 'Pattern editing: Make patterns "content only" by default upon insertion', 'gutenberg' ),
+			'description' => __( 'When patterns are inserted, default to a simplified content only mode for editing pattern content.', 'gutenberg' ),
+			'category'    => 'patterns',
+		),
+		array(
+			'id'          => 'gutenberg-content-only-inspector-fields',
+			'title'       => __( 'Pattern editing: Enable editable inspector fields', 'gutenberg' ),
+			'description' => __( 'Enables editable inspector fields (media, links, alt text, etc.) in the "content only" pattern editing interface. Requires "Pattern editing: Make patterns "content only" by default upon insertion" to be enabled.', 'gutenberg' ),
+			'category'    => 'patterns',
+			'requires'    => 'gutenberg-content-only-pattern-insertion',
+		),
+		array(
+			'id'          => 'gutenberg-workflow-palette',
+			'title'       => __( 'Workflow Palette', 'gutenberg' ),
+			'description' => __( 'Enables the Workflow Palette for running workflows composed of abilities, from a unified interface.', 'gutenberg' ),
+			'category'    => 'workflow',
+		),
+		array(
+			'id'          => 'gutenberg-customizable-navigation-overlays',
+			'title'       => __( 'Customizable Navigation Overlays', 'gutenberg' ),
+			'description' => __( 'Enables custom mobile overlay design and content control for Navigation blocks, allowing you to create flexible, professional menu experiences.', 'gutenberg' ),
+			'category'    => 'styling',
+		),
+	);
+}
+
+/*
+ * END CONFIGURATION.
+ * If you're not adding new experiments, you can stop here.
  */
 
 if ( ! function_exists( 'the_gutenberg_experiments' ) ) {
 	/**
 	 * The main entry point for the Gutenberg experiments page.
-	 *
-	 * @since 6.3.0
 	 */
 	function the_gutenberg_experiments() {
+		$experiments   = gutenberg_get_experiment_definitions();
+		$enabled_count = 0;
+		$total_count   = count( $experiments );
+		$options       = get_option( 'gutenberg-experiments', array() );
+
+		foreach ( $experiments as $experiment ) {
+			if ( isset( $options[ $experiment['id'] ] ) && $options[ $experiment['id'] ] ) {
+				++$enabled_count;
+			}
+		}
+
+		// Count active_templates separately
+		if ( gutenberg_is_experiment_enabled( 'active_templates' ) ) {
+			++$enabled_count;
+			++$total_count;
```

Comment:
Is it right that the `total_count` is only incremented when the experiment is enabled?